charts/wadm/Chart.yaml

```diff
@@ -15,10 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: "0.2.7"
+version: "0.2.8"
```

Comment:
I would say you can even cut a `0.3.0` for this if you'd like, I'm fine either way